### PR TITLE
[dashboard] remove deprecated tailwind-filter

### PR DESCRIPTION
**Pull request by flaming-codes**
## Description
Remove the depreacted **tailwind-filters** package and replace its usage w/ native tailwind utility classes.

## Related Issue(s)
Fixes https://github.com/gitpod-io/gitpod/issues/10004

## How to test
No specific test for these styles are available.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: components/dashboard/src/index.css [hunk 1]
```diff
@@ -40,9 +40,8 @@
     .btn-login {
         @apply rounded-md border-none bg-gray-100 hover:bg-gray-200 text-gray-500 dark:text-gray-200 dark:bg-gray-800 dark:hover:bg-gray-600;
     }
-    .dark .dark\:filter-invert {
-        @apply filter-invert;
-    }
```
Comment (flaming-codes) [inline components/dashboard/src/index.css:45]: Not required any more, as the setting to enable `dark:invert` is added to the `tailwind.config.js` w/ this PR.

**Comment by gtsiolis** [comment]
Looking at this now! 👀 

/werft run

**Review by gtsiolis — CHANGES_REQUESTED** [review]
Thanks for contributing, @flaming-codes! ✨ 

Changes look good! However, the filters from tailwind do not seem to work over SVG images. Wondering if this is a known issue since [`filter`] property is expected to work with SVGs, too. 💭 

| Login  | Login (Repository) | First time log in modal | Preferences |
|-|-|-|-|
| Screenshot 2022-05-26 at 3 40 47 PM (2) | Screenshot 2022-05-26 at 3 40 43 PM (2)  | Screenshot 2022-05-26 at 3 41 41 PM (2) | Screenshot 2022-05-26 at 3 41 51 PM (2) |

Let me spin up another preview environment instance below.

**Comment by gtsiolis** [comment]
You'll also need to sign a Contributor License Agreement (CLA)[[1](https://en.wikipedia.org/wiki/Contributor_License_Agreement)] once before merging your first contribution. Looping in [@meysholdt](https://github.com/meysholdt) to reach out about the CLA. 🏓

**Comment by flaming-codes** [comment]
Added missing utility class, as required by Tailwind 2.x. Whenever we upgrade to 3.x, the `filter`-utility can be removed.

| Login | Providers |
| --- | --- |
| Screenshot 2022-05-26 at 22 37 16 | Screenshot 2022-05-26 at 22 36 46 |

**Comment by flaming-codes** [comment]
@meysholdt I would really like to sign the CLA so that I can continue contributing to Gitpod. My email is tom@flaming.codes. Thanks!

**Comment by aboqasem** [comment]
Hi @flaming-codes, [here are the instructions on signing the CLA](https://github.com/gitpod-io/gitpod/blob/main/dev/addlicense/CONTRIBUTING.md).

**Comment by flaming-codes** [comment]
> Hi @flaming-codes, [here are the instructions on signing the CLA](https://github.com/gitpod-io/gitpod/blob/main/dev/addlicense/CONTRIBUTING.md).

I already signed the CLA, is there anything more to do? Thanks!

**Comment by flaming-codes** [comment]
@gtsiolis What's the status on this from your side? I signed the CLA a long time ago, is there anything to do on my part? Thanks!

**Comment by gtsiolis** [comment]
@flaming-codes Did you receive a CLA doc to sign from @meysholdt as described in https://github.com/gitpod-io/gitpod/pull/10268#issuecomment-1138589118? I've also asked the team internally to confirm.

**Comment by flaming-codes** [comment]
> @flaming-codes Did you receive a CLA doc to sign from @meysholdt as described in [#10268 (comment)](https://github.com/gitpod-io/gitpod/pull/10268#issuecomment-1138589118)? I've also asked the team internally to confirm.

Yes, and I signed it some weeks ago.

**Comment by meysholdt** [comment]
hi @flaming-codes, I checked, but we don't have a signed CLA from your on record. Maybe that is because the link from the comments above points to Googles CLA and not to Gitpods CLA. Apologies for the confustion. Can you please sign our CLA via [this DocuSign form](https://eu.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=30e1aa46-cce5-4bc6-a933-ad02cd8d5ada&env=eu&acct=86335c2c-01b9-4edc-a187-014f5150a112&v=2)? If there are any questions, you can reach me via moritz@gitpod.io.

**Comment by flaming-codes** [comment]
> hi @flaming-codes, I checked, but we don't have a signed CLA from your on record. Maybe that is because the link from the comments above points to Googles CLA and not to Gitpods CLA. Apologies for the confustion. Can you please sign our CLA via [this DocuSign form](https://eu.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=30e1aa46-cce5-4bc6-a933-ad02cd8d5ada&env=eu&acct=86335c2c-01b9-4edc-a187-014f5150a112&v=2)? If there are any questions, you can reach me via [moritz@gitpod.io](mailto:moritz@gitpod.io).

@meysholdt Great, thanks! I signed the doc provided by the link.

**Comment by meysholdt** [comment]
Awesome, thank you! I've received the signed CLA and filed a PR to remove the misleading instructions: https://github.com/gitpod-io/gitpod/pull/11646

**Comment by flaming-codes** [comment]
Due to the lack of action for a final review and me atm not having the necessary time to further work on it, I'm closing this PR.

**Pull request closed without merging**